### ISSUE TITLE
Searching algorithms

### ISSUE THREAD

**Issue by sumit-awasthi** [phumipatc/Algorithm#38]
I would like to add searching algorithms because it is not the part of your repo till now ... So shall I ?

if yes please assign me this issue..

**Comment by phumipatc** [comment]
yes, of course. You can add any interesting searching algorithm.

**Comment by sumit-awasthi** [comment]
Thanks

**Comment by Saurabh-2809** [comment]
Hi @phumipatc, I would also like to add the Fibonacci Search.
Could you please assign me this issue?

**Comment by srusti610** [comment]
hello sir/madam,
can I contribute to the issue.
